lib/src/mock-server/server.ts

```diff
@@ -72,7 +73,10 @@ export function runMockServer(
         return;
       }
     }
+
     logger.error(`No match for request ${req.method} at ${req.path}.`);
+    resp.status(404);
+    resp.send();
```

Comment:
Upon not finding a valid route, we still need to respond with a HTTP response. Add a 404 response.